js/views/PageNav.js

```diff
@@ -57,11 +58,44 @@ export default class PageNav extends View {
     }
   }
 
+  onKeyupAddressBar(e) {
+    if (e.which === 13) {
+      let text = this.$addressBar.val();
+
+      if (text.startsWith('ob://')) text = text.slice(5);
+
+      if (text.charAt(0) === '@' && text.length > 1) {
+        // a handle
+        app.router.navigate((text.split(' ')[0]), { trigger: true });
+      } else if (text.startsWith('Qm')) {
```

Comment:
For now, I'm assuming v2 guids we'll be prefaced with `Qm`. @cpacia did not seem to 100% guarantee this would be the case, but determining if a string is a valid base58 string is not a reliable way to distinguish between a guid and tag, since `shoes` is a valid base58 string (but obviously not a GUID).

So, I'm thinking we will need some prefix.